### ISSUE TITLE
fasterq-dump Syntax Ambiguous

### ISSUE THREAD

**Issue by DarioS** [ncbi/sra-tools#239]
The usage is described as

```
Usage:
  fasterq-dump <path> [options]
```

where should a sample ID such as SRR5790104 be placed? It's not an option and it's not a file path. Come to think of it, `path` isn't defined below the Usage section but perhaps it should explicitly be.

**Comment by klymenko** [comment]
Yes, the message is confusing. We will fix it in our next release.
Use accession as command line argument: `fasterq-dump SRR5790104`.

**Comment by klymenko** [comment]
@DarioS, it was fixed.
Please check our latest release.
